File: catalogue-api/src/test/java/io/quartic/catalogue/api/CloudGeoJsonDatasetLocatorShould.java
package io.quartic.catalogue.api;

public class CloudGeoJsonDatasetLocatorShould extends DatasetLocatorTests<CloudGeoJsonDatasetLocator> {
    @Override
    protected CloudGeoJsonDatasetLocator locator() {
        return CloudGeoJsonDatasetLocatorImpl.of("http://wat", false);
    }

    @Override
    protected String json() {
        return "{\"type\": \"cloud-geojson\", \"path\": \"http://wat\"}";
    }
}

File: catalogue-api/src/test/kotlin/io/quartic/catalogue/api/CloudGeoJsonDatasetLocatorShould.kt
package io.quartic.catalogue.api

class CloudGeoJsonDatasetLocatorShould : DatasetLocatorTests<CloudGeoJsonDatasetLocator>() {
    override fun locator() = CloudGeoJsonDatasetLocator("http://wat", false)

    override fun json() = "{\"type\": \"cloud-geojson\", \"path\": \"http://wat\"}"
}
